test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlDocumentTest.cs

```diff
@@ -70,7 +70,11 @@ public void ParseDocumentCorrectlyHandlesOddlySpacedHTMLElements()
             ParseDocumentTest("<div ><p class = 'bar'> Foo </p></div >",
                 new MarkupBlock(
                     BlockFactory.MarkupTagBlock("<div >"),
-                    BlockFactory.MarkupTagBlock("<p class = 'bar'>"),
+                    new MarkupTagBlock(
+                        Factory.Markup("<p"),
+                        new MarkupBlock(
+                            Factory.Markup(" class")),
+                        Factory.Markup(" = 'bar'>")),
```

Comment:
Razor today doesn't understand space between `=`. Therefore, now that the parser understands minimized attributes (but still not space between `=`, that's another issue) it thinks `class` is a minimized attribute.